### PR TITLE
BDRSPS-925 Add collection for sensitivityCategory attribute

### PR DESCRIPTION
**Pull request by Lincoln-GR**
The `sensitivityCategory` attribute node is currently "floating" (not attached to anything)
This PR add a collection for it which is also floating. Once `abis:BiodiversityRecord` is implemented, the collection will be linked to that.

### CODE REVIEW

File: abis_mapping/templates/incidental_occurrence_data_v3/mapping.py [hunk 1]
```diff
@@ -3641,24 +3663,23 @@ def add_conservation_authority_collection(
...
+        sensitivity_category_value: rdflib.URIRef | None,
         graph: rdflib.Graph,
     ) -> None:
         """Adds Sensitivity Category Attribute to the Graph
 
         Args:
-            uri (rdflib.URIRef): URI to use for this node.
+            uri: URI to use for this node.
             row (frictionless.Row): Row to retrieve data from
             dataset (rdflib.URIRef): Dataset this belongs to
```
Comment (joecrowleygaia) [inline abis_mapping/templates/incidental_occurrence_data_v3/mapping.py:3677]: Disagree if you want, but i think if we omit one arg type from a docstring then it should be omitted for all in that block. I'm basing this on the loose statement within the google guide of being "consistent".

File: abis_mapping/templates/incidental_occurrence_data_v3/mapping.py [hunk 2]
```diff
@@ -3668,25 +3689,26 @@ def add_sensitivity_category_attribute(
...
         graph: rdflib.Graph,
     ) -> None:
         """Adds Sensitivity Category Value to the Graph
 
         Args:
-            uri (rdflib.URIRef): URI to use for this node
+            uri: URI to use for this node
             row (frictionless.Row): Row to retrieve data from
             dataset (rdflib.URIRef): Dataset this belongs to
             graph (rdflib.Graph): Graph to add to
```
Comment (joecrowleygaia) [inline abis_mapping/templates/incidental_occurrence_data_v3/mapping.py:3708]: remove types (maybe)

File: abis_mapping/templates/survey_occurrence_data_v2/mapping.py [hunk 3]
```diff
@@ -4358,24 +4380,23 @@ def add_site(
...
+        sensitivity_category_value: rdflib.URIRef | None,
         graph: rdflib.Graph,
     ) -> None:
         """Adds Sensitivity Category Attribute to the Graph
 
         Args:
-            uri (rdflib.URIRef): URI to use for this node.
+            uri: URI to use for this node.
             row (frictionless.Row): Row to retrieve data from
             dataset (rdflib.URIRef): Dataset this belongs to
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_occurrence_data_v2/mapping.py:4394]: remove types (maybe)

File: abis_mapping/templates/survey_occurrence_data_v2/mapping.py [hunk 4]
```diff
@@ -4358,24 +4380,23 @@ def add_site(
...
-            sensitivity_category_value (rdflib.URIRef): Sensitivity
-                Category Value associated with this node
+            sensitivity_category_value: Sensitivity Category Value associated with this node
             graph (rdflib.Graph): Graph to add to
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_occurrence_data_v2/mapping.py:4396]: remove type (maybe)

File: abis_mapping/templates/survey_occurrence_data_v2/mapping.py [hunk 5]
```diff
@@ -4385,25 +4406,26 @@ def add_sensitivity_category_attribute(
...
         graph: rdflib.Graph,
     ) -> None:
         """Adds Sensitivity Category Value to the Graph
 
         Args:
-            uri (rdflib.URIRef): URI to use for this node
+            uri: URI to use for this node
             row (frictionless.Row): Row to retrieve data from
             dataset (rdflib.URIRef): Dataset this belongs to
             graph (rdflib.Graph): Graph to add to
```
Comment (joecrowleygaia) [inline abis_mapping/templates/survey_occurrence_data_v2/mapping.py:4425]: remove types (maybe)

**Review by joecrowleygaia — COMMENTED** [review]
All good just a question regarding mixing inclusion/omission of types in docstrings.

File: abis_mapping/templates/incidental_occurrence_data_v3/mapping.py (hunk 1 above)
Comment (Lincoln-GR) [reply abis_mapping/templates/incidental_occurrence_data_v3/mapping.py:3677]: Yes good point. I was operating on "change least number of lines" mode. making this consistent for a function is sensible.

**Review by joecrowleygaia — APPROVED** [review]

**Pull request merged**